framework/Dockerfile

```diff
@@ -34,6 +33,30 @@ ENV IDRINTH_API_BENCH_VERSION=$BUILD_VERSION
 ENV IDRINTH_API_BENCH_BUILT=$BUILD_TIME
 
 RUN mkdir -p /usr/local/lib/node_modules/@idrinth/api-bench
```

Comment:
why use the old name? the current one would be @idrinth-api-bench/framework

Reply:
added framework